### PR TITLE
addresses https://github.com/gitcoinco/web/issues/10762

### PR DESCRIPTION
**Pull request by owocki**
##### Description

addresses https://github.com/gitcoinco/web/issues/10762 by removing the hackathon object from the reuslts page

##### Refers/Fixes

https://github.com/gitcoinco/web/issues/10762

##### Testing

tested locally

### CODE REVIEW

File: app/perftools/management/commands/create_page_cache.py [hunk 1]
```diff
@@ -329,6 +329,7 @@ def create_results_cache():
             print(f"- executing {keyword}")
             data = build_stat_results(keyword)
             print("- creating")
+            del data['hackathons']
```
Comment (nutrina) [inline app/perftools/management/commands/create_page_cache.py:332]: Just to be on the safe side:

```suggestion
            if 'hackathons' in data:
                    del data['hackathons']
```

**Review by nutrina — CHANGES_REQUESTED** [review]
I would fix one minor thing ...

**Comment by owocki** [comment]
@nutrina  just commited your patch. my first time using the 'propposed changes' feature. quite nice!

**Review by nutrina — APPROVED** [review]
lgtm

**Pull request merged**